.flowconfig

```diff
@@ -29,7 +29,6 @@
 ; https://flow.org/en/docs/config/lints/
 [lints]
 all=error
-ambiguous-object-type=off
```

Comment:
Is this ON by default now?

Reply:
I am a little bit confused here, I had to remove it from strict, but I can see it gives no errors when it is off, and no errrors when on. 

On the strict config I got errors 🤔

Reply:
It says `Like implicit-inexact-object, except triggers even when the exact_by_default option is set to true.`

Which seems like it does not, so, I can leave it to off